source/_docs/z-wave/device-specific.markdown

```diff
@@ -49,6 +49,24 @@ echo -e -n "...turn on/off string from examples above..." | cu -l /dev/zstick -s
 
 ### Razberry Board
 
+<div class='note warning'>
```

Comment:
Pure style note:

This should not all be in a note. This should be a new section called "Razberry Board with Hass.io".